GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributors.cs

```diff
@@ -12,21 +12,21 @@ namespace GitFyle.Core.Api.Brokers.Storages
 {
     internal partial class StorageBroker
     {
-        public DbSet<Contributor> Contributors {  get; set; }
+        public DbSet<Contributor> Contributors { get; set; }
 
         public ValueTask<Contributor> InsertContributorAsync(Contributor contributor) =>
-            InsertAsync(contributor); 
+            InsertAsync(contributor);
```

Comment:
Need to make this async

Reply:
I worked on Configuration Object. Shall I do changes to broker related to Contributor object?

Reply:
Done.